pkg/storageos/scheduler_extender.go

```diff
@@ -66,14 +66,11 @@ func (s Deployment) createSchedulerDeployment(replicas int32) error {
 		},
 	}
 
-	// Add cluster config tolerations.
-	if err := s.addTolerations(&spec.Template.Spec); err != nil {
+	// Add helper tolerations.
+	if err := s.addHelperTolerations(&spec.Template.Spec, podTolerationSeconds); err != nil {
```

Comment:
Looks like `addCommonPodProperties()` is not added to the scheduler pod. It exists in api-manager and csi-helpers. We can replace `addHelperTolerations()` with `addCommonPodProperties()` to add toleration, affinity and any other common attributes.

Reply:
yes, it seemed odd that the scheduler wasn't applying `addCommonPodProperties()`.  Added it.